tool/src/java/org/sakaiproject/attendance/tool/pages/BasePage.java

```diff
@@ -173,6 +175,7 @@ public void onClick() {
 		this.addOrEditItemWindow.showUnloadConfirmation(false);
 		this.addOrEditItemWindow.setInitialHeight(400);
 		add(this.addOrEditItemWindow);
+		Session.get().setLocale((new ResourceLoader()).getLocale());
```

Comment:
There is already a method to getLocale in sakaiProxy? Should we remove that ? 

sakaiProxy.getCurrentUserLocale()

Reply:
Also, I see this pattern in other Sakai files:

getSession().setLocale() 

instead of Session.get()